lib/rules/web/admin_console/4.2/catalog.xyaml

```diff
@@ -56,3 +56,40 @@ clear_filter_by_keyword:
   element:
     selector: *filter_by_keyword
     op: clear
+click_on_catalog_item:
+  element:
+    selector:
+      xpath: //a[@data-test='<operator_name>-redhat-operators-openshift-marketplace']
```

Comment:
@yapei  I think it's better to use `<opsrc_name>` instead of `redhat-operators` because in our testing, we always use the operator source created by ourself not the default one.

Reply:
good point, will update